packages/store-indexer/src/postgres/getLogs.ts

```diff
@@ -53,7 +53,8 @@ export async function getLogs(
   const records = await database
     .select()
     .from(tables.recordsTable)
-    .where(or(...conditions));
+    .where(or(...conditions))
+    .orderBy(asc(tables.recordsTable.lastUpdatedBlockNumber));
```

Comment:
originally had this as 
```ts
.orderBy(sql`${tables.recordsTable.lastUpdatedBlockNumber} ASC NULLS FIRST`)
```
but then realize there's ~no case where we'd add records to the table _without_ a block number (both RPC and another indexer will provide this info)